applications/StructuralMechanicsApplication/python_scripts/impose_rigid_movement_process.py

```diff
@@ -0,0 +1,108 @@
+import KratosMultiphysics as KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ImposeRigidMovementProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class ImposeRigidMovementProcess(KratosMultiphysics.Process):
+    """This class is used in order to impose a rigid body movement in a certain region of the problem
+
+    This class constructs the model parts containing the constrains that enforce the rigid body movement
+    Only the member variables listed below should be accessed directly.
+
+    Public member variables:
+    Model -- the container of the different model parts.
+    settings -- Kratos parameters containing solver settings.
+    """
+
+    def __init__(self, Model, settings ):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing solver settings.
+        """
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "help"                        : "This process uses LinearMasterSlaveConstraint in order to impose an unified movement in the given submodelpart. The process takes the first node from the submodelpart if no node's ID is provided. The default variable is DISPLACEMENT, and in case no variable is considered for the slave the same variable will be considered",
+            "computing_model_part_name"   : "computing_domain",
+            "model_part_name"             : "please_specify_model_part_name",
+            "new_model_part_name"         : "",
```

Comment:
This "new_model_part_name" can be gone. It just is not clear to follow why is this required and it is easy to get confused with different modelparts here ... 
Can you leave a comment in the code before merging ?

Reply:
It is empty by default, and some people->me, likes to add the constraints to a differerent model part